test/utils/wabt_engine.cpp

```diff
@@ -66,7 +66,7 @@ bool WabtEngine::instantiate(bytes_view wasm_binary)
     const auto host_func = wabt::interp::HostFunc::New(m_store,
         wabt::interp::FuncType{{wabt::Type::I32, wabt::Type::I32}, {wabt::Type::I32}},
         [](wabt::interp::Thread& thread, const wabt::interp::Values& args,
-            wabt::interp::Values& results, wabt::interp::Trap::Ptr*) -> wabt::Result {
+            wabt::interp::Values& results, wabt::interp::Trap::Ptr*) noexcept -> wabt::Result {
```

Comment:
Maybe this is not a good ideas as wabt helpers may throw?

Reply:
Not sure. Do they?

Reply:
`GetCallerInstance()` indexes a vector (with local bounds checking), `memories()`, `UnsafeData` and `ByteSize` seems fine. So I guess it is ok.